### PR TITLE
Make aws credentials file uniformly named

### PR DESCRIPTION
**Pull request by shyamjvs**
This is the name also used by the aws-sdk.

/cc @gyuho @krzyzacy

### CODE REVIEW

**Comment by shyamjvs** [comment]
FYI - This will allow using different testing accounts for kops/eks tests (by just varying the preset). I'll send a follow-up PR soon.

File: experiment/bootstrap/main.go [hunk 1]
```diff
@@ -30,8 +30,8 @@ const (
 	CloudSDKEnv            string = "CLOUDSDK_CONFIG"
 	GCEPrivKeyEnv          string = "JENKINS_GCE_SSH_PRIVATE_KEY_FILE"
 	GCEPubKeyEnv           string = "JENKINS_GCE_SSH_PUBLIC_KEY_FILE"
-	AWSPrivKeyEnv          string = "JENKINS_AWS_SSH_PRIVATE_KEY_FILE"
-	AWSPubKeyEnv           string = "JENKINS_AWS_SSH_PUBLIC_KEY_FILE"
+	AWSPrivKeyEnv          string = "AWS_SSH_PRIVATE_KEY_FILE"
```
Comment (BenTheElder) [inline experiment/bootstrap/main.go]: heh reminder to myself delete this experiment. we're definitely not going this route

**Comment by shyamjvs** [comment]
Sure.

**Comment by krzyzacy** [comment]
Ummm, who's consuming these envs downstream? Seems not in test-infra? cc @justinsb who might know?

**Comment by shyamjvs** [comment]
@krzyzacy AWS SDK uses it (see vendor).

**Pull request merged**